### PR TITLE
[IMPROVEMENT] Refactor Personal Signature

### PR DESCRIPTION
**Pull request by blackdevelopa**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**
Refactor the personal sign code from JavaScript to Typescript

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses NA

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/UI/PersonalSign/PersonalSign.tsx [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    messageTextColor: {
+      color: colors.text.default,
+    },
+    textLeft: {
+      textAlign: 'left',
+    },
+    messageWrapper: {
+      marginBottom: 4,
+    },
+  });
```
Comment (gantunesr) [inline app/components/UI/PersonalSign/PersonalSign.tsx:38]: Move this to a `styles.ts` file

File: app/components/UI/PersonalSign/PersonalSign.tsx [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        dapp_host_name: url?.host,
+        dapp_url: currentPageInformation?.url,
+        chain_id: chainId,
+        sign_type: 'personal',
+        ...currentPageInformation?.analytics,
+      };
+    } catch (error) {
+      return {};
+    }
+  }, [currentPageInformation, selectedAddress]);
```
Comment (gantunesr) [inline app/components/UI/PersonalSign/PersonalSign.tsx:79]: Do you think it would be better for this method to have a return type?

**Review by gantunesr — CHANGES_REQUESTED** [review]

**Comment by gantunesr** [comment]
Great work! left some feedback

File: app/components/UI/PersonalSign/PersonalSign.tsx (hunk 1 above)
Comment (blackdevelopa) [reply app/components/UI/PersonalSign/PersonalSign.tsx]: https://github.com/MetaMask/metamask-mobile/pull/5551/commits/48a8a3180ae4d6a6be1b7b3d810b0296d7110082

**Review by gantunesr — APPROVED** [review]
LGTM

**Pull request merged**